src/olympia/templates/mobile/base_addons.html

```diff
@@ -89,6 +94,9 @@ <h1 class="site-title">
     {% block site_js %}
       <script src="{{ cache_buster(url('jsi18n')) }}"></script>
       {{ js('zamboni/mobile') }}
+      {% if switch_is_active('fxa-auth') %}
```

Comment:
I'd rather see you add all fxa JS to the [zamboni/mobile bundle](https://github.com/mozilla/addons-server/blob/master/src/olympia/lib/settings_base.py#L849) because making a new http request for this bundle is going to hurt.